gustaf/create/volumes.py

```diff
@@ -35,3 +35,155 @@ def box(bounds=[[0., 0., 0.], [1., 1., 1.]], resolutions=[2, 2, 2]):
     volume_mesh = Volumes(vertex_mesh.vertices, connectivity)
 
     return volume_mesh
+
+def extrude_to_tet(
```

Comment:
I'd much prefer bringing back a wrapper to tetgen. Could you please remove this part for this PR?